components/alert/test/alert.visual-diff.js

```diff
@@ -20,7 +20,6 @@ describe('d2l-alert', () => {
 		'type-default',
 		'type-success',
 		'type-critical',
-		'type-warning',
```

Comment:
Might be worth adding a few RTL visual diff tests to make sure we don't forget about RTL in the future.

Reply:
[There is one ](https://github.com/BrightspaceUI/core/blob/d33e4be8995142432edd303412f3d35fd606685d/components/alert/test/alert.visual-diff.js#L28) that covers the regular differences, I think I missed it because the padding changes meant all of the v-diff tests were failing either way (and for some reason when I run the tests locally I get the 'before' and 'after' pictures, but not the one that highlights what the difference is). I'm not sure if there's a second case that would be meaningful if added 🤔 open to ideas, though.

What I notice we don't have is a test for what happens on a small-width viewport. I can't recall seeing tests for that elsewhere - are the special spec'd adjustments for the toasts a fairly unique case or is the tiny screen test something we don't usually do?

Reply:
We typically only do it if there are responsive rules to test. Here's [dialog's example](https://github.com/BrightspaceUI/core/blob/master/components/dialog/test/dialog.visual-diff.js#L43) of it.

Reply:
For dialogs (and other things) I tried to focus the responsive tests on that would be different.  i.e. there are quite a few tests, and it wasn't really necessary or good use of build-time to repeat them all for both wide & narrow.

Reply:
Given that the only changes are the text size and the distance from the edge of the screen in the case of toasts, I probably only need to have one extra test for this. The other tests here are mostly colors and internal elements that shouldn't need to be duped. :)